scripts/install-windows-prereqs.ps1

```diff
@@ -2,6 +2,7 @@
 # Licensed under the MIT License.
 
 Param(
+    [Parameter(Mandatory=$true)][string]$installPath,
```

Comment:
This will break the Ansible openenclave/scripts/ansible/oe-windows-acc-setup.yml and that needs to be updated as well.

Reply:
What should be used for Ansible?

Reply:
Kind of, I would use the original hardcoded value: `${env:SystemDrive} "openenclave"`

Reply:
I think is best to NOT make the parameter required in order to avoid the ansible specify the path.  I will instead default it in the script.

Reply:
Resolved.